ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -328,6 +328,35 @@ class FamilyReads extends React.PureComponent {
     )
   }
 
+  getSampleColors = () => {
+    const { openFamily } = this.state
+    const { igvSamplesByFamilySampleIndividual, individualsByGuid } = this.props
+    const igvSampleIndividuals = (
+      openFamily && (igvSamplesByFamilySampleIndividual || {})[openFamily]) || {}
+    return Object.keys(igvSampleIndividuals[GCNV_TYPE] || {}).reduce(
+      (acc, iGuid) => ({
+        ...acc,
+        [iGuid]: individualsByGuid[iGuid].affected === AFFECTED ? 'red' : 'blue',
+      }), {},
+    )
+  }
+
+  getSampleColorPanel = () => {
+    const { individualsByGuid } = this.props
+    const sampleColors = this.getSampleColors()
```

Comment:
This code works, but the control flow is a little confusing. I think it might be easier to understand if you simply get a list of the gcnv individuals and then loop through those and display them. Something like
```
const gcnvIndividuals = Object.keys(igvSampleIndividuals[GCNV_TYPE] || {}).map(iGuid => individualsByGuid[iGuid])
return gcnvIndividuals.map(individual => (
 <div>
 <Icon name="square full" color={getSampleColor(individual)} />
<label>{individual.displayName}</label>
...
))
```